"long unsigned int" (format: "lu") and "uint64_t" don't match on i586. Size of
"long unsigned int" is only 32-bit on i586.

Use the appropriate format for uint64_t: PRIu64.
--
diff -up cuishark-0.1.2/PacketListPane.cc.orig cuishark-0.1.2/PacketListPane.cc
--- cuishark-0.1.2/PacketListPane.cc.orig	2019-08-17 21:44:46.672894062 +0200
+++ cuishark-0.1.2/PacketListPane.cc	2019-08-17 22:05:40.600946047 +0200
@@ -1,4 +1,4 @@
-
+#include <inttypes.h>
 #include "PacketListPane.h"
 #include "protocol.h"
 #include "TextPane.h"
@@ -153,11 +153,7 @@ std::string Packet::to_str() const
 {
   using namespace slankdev;
   char sstr[1000];
-#if __APPLE__
-  sprintf(sstr, "%5zd %-13llu %-20s %-20s %-6s %5zd %-10s" , number, time,
-#else
-  sprintf(sstr, "%5zd %-13lu %-20s %-20s %-6s %5zd %-10s" , number, time,
-#endif
+  sprintf(sstr, "%5zd %-13" PRIu64 "%-20s %-20s %-6s %5zd %-10s" , number, time,
           src.c_str(),
           dst.c_str(),
           protocol.c_str(), len,
